nilearn/masking.py

```diff
@@ -20,9 +20,6 @@ class MaskWarning(UserWarning):
     "A class to always raise warnings"
 
 
-warnings.simplefilter("always", MaskWarning)
```

Comment:
Can you motivate this one? It seems like you are changing the warning policy significantly but maybe you have a good reason for doing so.